packages/graphql/src/schema-model/entity/model-adapters/InterfaceEntityAdapter.ts

```diff
@@ -53,44 +53,11 @@ export class InterfaceEntityAdapter {
         this.initConcreteEntities(entity.concreteEntities);
     }
 
-    public findAttribute(name: string): AttributeAdapter | undefined {
```

Comment:
This bit here is just reordering some methods in InterfaceEntityAdapter to make it more consistent